revscoring/features/wikitext/datasources/edit.py

```diff
@@ -0,0 +1,312 @@
+import logging
+import re
+import time
+
+from deltas import segment_matcher
+
+from ....datasources import Datasource
+from ....datasources.meta import filters
+from .tokenized import TokenIsInTypes, is_uppercase_word
+
+logger = logging.getLogger(__name__)
+
+
+class Diff:
```

Comment:
As discussed in chat, this is only necessary python 2.x.  In python 3, inheritance from `object` is implied.